rules/macos/defense_evasion_attempt_del_quarantine_attrib.toml

```
@@ -17,16 +17,27 @@ license = "Elastic License"
 name = "Attempt to Remove File Quarantine Attribute"
 references = [
     "https://www.trendmicro.com/en_us/research/20/k/new-macos-backdoor-connected-to-oceanlotus-surfaces.html",
+    "https://ss64.com/osx/xattr.html",
 ]
-risk_score = 43
+risk_score = 47
 rule_id = "f0b48bbc-549e-4bcf-8ee0-a7a72586c6a7"
 severity = "medium"
 tags = ["Elastic", "Host", "macOS", "Threat Detection", "Defense Evasion"]
 type = "eql"
 
 query = '''
 process where event.type in ("start", "process_started") and
-  process.name == "xattr" and process.args == "com.apple.quarantine" and process.args == "-d"
+  process.args : "xattr" and
+  (
+    (process.args : "com.apple.quarantine" and process.args : ("-d", "-w")) or
+    (process.args : "-c" and process.command_line :
+      (
+        "/bin/bash -c xattr -c *",
+        "/bin/zsh -c xattr -c *",
+        "/bin/sh -c xattr -c *"
+      )
+    )
+  )
```

Could we simplify this with

```suggestion
    (process.args : "-c" and process.args_count > 4
    )
  )
```

Is `process.args_count` in both `auditbeat-*` and `logs-endpoint.events.*`. This would also eliminate a wildcard

---

@brokensound77 this will unfortunately match also on below command (unrelated to clearing extended attrib), on macOS observed that xattr is often not logged as a process event, instead its logged as args of a shell process event:

```
/bin/sh -c xattr -l file.app
```

---

ah ok, good point, thanks